packages/pn-personafisica-webapp/src/pages/Contacts.page.tsx

```diff
@@ -100,13 +100,13 @@ const Contacts = () => {
           reloadAction={fetchAddresses}
         >
           <ContactsSummaryCards />
-          <Stack direction="column" spacing={4}>
+          <Stack direction="column" spacing={6}>
             <LegalContacts />
             <CourtesyContacts />
           </Stack>
           {showSpecialContactsSection && (
             <>
-              <Divider sx={{ backgroundColor: 'white', color: 'text.secondary', mt: 6, mb: 3 }} />
+              <Divider sx={{ backgroundColor: 'white', color: 'text.secondary', mt: 6, mb: 4 }} />
```

Comment:
The color of divider should be: "#A2ADB8" (text.disabled)

Reply:
I think you have to modify the value of border color instead of color

Reply:
color is #E3E7EB in the figma sheet